tests/e2e/common/common.go

```diff
@@ -244,7 +247,7 @@ func StatusTestOnInstallUpgrade(details VersionDetails, opts TestOptions) func(t
 				"dapr-operator":         {details.RuntimeVersion, "3"},
 			}
 		}
-
+		t.Log(output)
```

Comment:
is this needed ?

Reply:
Not really, just added to get actual status.

Reply:
please remove ...